dirty_cat/datasets/_ken_embeddings.py

```diff
@@ -0,0 +1,97 @@
+"""
+Get the Wikipedia embeddings for feature augmentation.
+"""
+import pandas as pd
+from sklearn.decomposition import PCA
+
+from dirty_cat.datasets import fetch_figshare
+
+
+def get_ken_embeddings(
+    types,
+    exclude=None,
+    pca_components=None,
+    emb_id="39142985",
+    emb_type_id="39266300",
+    suffix="",
+):
+    """Extract Wikipedia embeddings by type.
+
+    More details on the embeddings can be found on
+    https://soda-inria.github.io/ken_embeddings/.
+
+    Parameters
+    ----------
+    types : str
+        Types of embeddings to include. Write in lowercase.
```

Comment:
It would be good to give a list of acceptable types, or a pointer to where this list can be found (eg by adding it on the ken embedding page)

Reply:
I will just leave for now a link to where the detailed types can be found, and then maybe we can improve this in a second iteration.